src/styles/reset.css

```diff
@@ -0,0 +1,23 @@
+/* #region reset */
+* {
+  box-sizing: border-box;
+}
+  
+html {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  font-style: normal;
+}
+  
+body,
+h2,
+p,
+.stars {
+  margin: 0;
+  padding: 0;
```

Comment:
these elements don't have default `padding`. why do you need to reset this for `.stars`?